runtime/tests/clinkerffi/downcall.c

```diff
@@ -4669,3 +4669,155 @@ addDoublesFromMultipleStructPtrs_returnStruct(stru_2_Doubles *arg1, stru_2_Doubl
 	doubleStruct.elem2 = arg1->elem2 + arg2->elem2 + arg3->elem2;
 	return doubleStruct;
 }
+
+/**
+ * Do xor operation on each boolean of a native array (dereferenced from a pointer to a native array)
+ * and assign the result to the element of a on-heap array at the same index.
+ *
+ * @param size the array size
+ * @param heapPtr a pointer to a on-heap array
+ * @param nativePtr a pointer to a native array
+ * @return the sum of integers
```

Comment:
Please correct documentation such as this (this function doesn't return anything).
There should be no `@return` for functions whose return type is `void`.

Reply:
Fixed.

Reply:
I still see several lines like
```
 * @return void
```
e.g. line 4680, 4699, 4718, etc.